### PR TITLE
Use CMS mgmt command for course content.

### PR DESCRIPTION
**Pull request by brianhw**
Also use CMS settings.  
Also removes support for discussion links, which is no longer used.
@mulby @HassanJaveed84

### CODE REVIEW

**Comment by brianhw** [comment]
Note that this depends on https://github.com/edx-ops/analytics-secure/pull/80 and a similar PR in edx-platform to define export_olx.

File: exporter/tasks.py [hunk 1]
```diff
@@ -809,13 +809,6 @@ def get_filename(cls, **kwargs):
         return os.path.join(kwargs['work_dir'], filename)
 
 
-class DiscussionLinkTask(CourseTask, DjangoAdminTask):
-    NAME = 'discussion_link'
-    COMMAND = 'get_discussion_link'
-    ARGS = '{course}'
-    OUT = '{filename}'
```
Comment (mulby) [inline exporter/tasks.py:816]: Why is this removed?

Comment (brianhw) [inline exporter/tasks.py:816]: We had removed the support for exporting the Berkeley discussion dumps (the name escapes me right now).  This is a task that existed to support that other task, and is not run.  So I thought I would remove it while in the code.

Comment (mulby) [inline exporter/tasks.py:816]: Got it, sounds good.

**Comment by mulby** [comment]
:+1:
